graphein/ppi/config.py

```diff
@@ -0,0 +1,94 @@
+    evidenceList: Optional[
+        str
+    ] = ""  # Any interaction evidence with its Experimental System in the list will be excluded from the results unless includeEvidence is set to true.
+    includeEvidence: Optional[
+        bool
+    ] = False  # If set to true, any interaction evidence with its Experimental System in the evidenceList will be included in the result
+    searchIds: Optional[
+        bool
+    ] = True  # If ‘true’, the interactor ENTREZ_GENE, ORDERED LOCUS and SYSTEMATIC_NAME (orf) will be examined for a match with the geneList.
+    searchNames: Optional[
+        bool
+    ] = True  # If ‘true’, the interactor OFFICIAL_SYMBOL will be examined for a match with the geneList.
+    searchSynonyms: Optional[
+        bool
+    ] = True  # If ‘true’, the interactor SYNONYMS will be examined for a match with the geneList.
+    searchBiogridIds: Optional[
+        bool
+    ] = True  # If ‘true’, the entries in 'GENELIST' will be compared to BIOGRID internal IDS which are provided in all Tab2 formatted files.
+    additionalIdentifierTypes: Optional[
+        str
+    ] = ""  # Identifier types on this list are examined for a match with the geneList.
+    excludeGenes: Optional[
+        bool
+    ] = False  # If ‘true’, interactions containing genes in the geneList will be excluded from the results.
+    includeInteractors: Optional[
+        bool
+    ] = True  # If ‘true’, in addition to interactions between genes on the geneList, interactions will also be fetched which have only one interactor on the geneList
+    includeInteractorInteractions: Optional[
+        bool
+    ] = False  # If ‘true’ interactions between the geneList’s first order interactors will be included.
+    pubmedList: Optional[
+        str
+    ] = ""  # Interactions will be fetched whose Pubmed Id is/ is not in this list, depending on the value of excludePubmeds.
+    excludePubmeds: Optional[
+        bool
+    ] = False  # If ‘false’, interactions with Pubmed ID in pubmedList will be included in the results; if ‘true’ they will be excluded.
+    htpThreshold: Optional[
+        int
+    ] = 20  # Interactions whose Pubmed ID has more than this number of interactions will be excluded from the results. Ignored if excludePubmeds is ‘false’.
+    throughputTag: Optional[
+        str
+    ] = "any"  # If set to 'low or 'high', only interactions with 'Low throughput' or 'High throughput' in the 'throughput' field will be returned.
+
+
+class PPIGraphConfig(BaseModel):
+    paginate: bool = True
+    ncbi_taxon_id: int = 9606
+    kwargs: Optional[Dict[str, Union[str, int, float]]] = {
+        "STRING_escore": 0.2,  # Keeps STRING interactions with an experimental score >= 0.2
+        "BIOGRID_throughputTag": "high",  # Keeps high throughput BIOGRID interactions
+    }
+    string_config: Optional[STRINGConfig] = None
+    biogrid_config: Optional[BioGridConfig] = None
+
+    class Config:
+        arbitrary_types_allowed: bool = True
```

Comment:
First time i've seen this pattern - how would we access/set the `arbitrary_types_allowed` attribute?

Reply:
We don't need to change this attribute. This is just for PyDantic to allow:

```
    string_config: Optional[STRINGConfig] = None
    biogrid_config: Optional[BioGridConfig] = None
```

Reply:
Aha! I learned something new today :smile: